Doppler.Sap/Enums/SapTaskEnum.cs

```diff
@@ -2,9 +2,8 @@ namespace Doppler.Sap.Enums
 {
     public enum SapTaskEnum
     {
-        CreateBusinessPartner = 1,
         BillingRequest = 2,
         CurrencyRate = 3,
-        UpdateBusinessPartner = 4
+        CreateOrUpdateBusinessPartner = 4
```

Comment:
Why CreateOrUpdateBussinesPartner is 4 and not 1?

Reply:
This is only because I modified UpdateBussinesPartner. I will change it!